requirements.txt

```diff
@@ -1 +0,0 @@
-pandas
```

Comment:
as per https://github.com/widgetti/ipyvuetify/issues/245, binder seems broken. I checked on my side and it's not starting either. What if we take advantage of this situation and create a extra-require[binder] instead of having a lonely requirements.txt ?

Reply:
We still want it to work when that issue is fixed. I didn't know about `extra-require[binder]`! I can't find that this is installed by binder in their docs though. It's still an unrelated change to this PR.

Reply:
solved in https://github.com/widgetti/ipyvuetify/pull/259/commits/f996604a71cfd00e913d9bd24d3d4f2451885aef